### ISSUE TITLE
How to write hi resolution scroll wheel events?

### ISSUE THREAD

**Issue by runekaagaard** [gvalkov/python-evdev#118]
First, thanks a lot for python-evdev!

I'm building a small linux "driver" for the Surface Dial over at https://github.com/runekaagaard/surface-dial-ubuntu-gnome. I'm trying to write hi res scrolling events with `REL_WHEEL_HI_RES`, but nothing happens. I've made a minimal failing example below. It works when using `REL_WHEEL`. Please help?

I'm on Ubuntu 19.10, Gnome, using the 5.3.18 kernel from https://github.com/linux-surface/linux-surface/releases on a Surface Book 2.

```
from evdev import UInput, ecodes as e, InputEvent
from time import sleep

cap = {
    e.EV_REL: [e.REL_WHEEL, e.REL_WHEEL_HI_RES],
}
ev = InputEvent(1334414993, 274296, e.EV_REL, e.REL_WHEEL_HI_RES, -1000)

with UInput(cap) as ui:
    while True:
        ui.write_event(ev)
        ui.syn()
        sleep(1)
```

**Comment by jonasBoss** [comment]
This has nothing to do with python-evdev. We encountered the same issue with https://github.com/sezanzeb/input-remapper/issues/354 
TLDR: WHEEL_HI_RES events are not processed by all Desktop environments. According to the [kernel documentation](https://www.kernel.org/doc/html/latest/input/event-codes.html?highlight=wheel_hi_res#ev-rel) 120 `REL_WHEEL_HI_RES` units is the same as 1 `REL_WHEEL` unit.

[input-remapper](https://github.com/sezanzeb/input-remapper) sends a `REL_WHEEL` event whenever `REL_WHEEL_HI_RES` accumulate to 120

**Comment by runekaagaard** [comment]
Good to know, THX!
